.github/workflows/docker-build-static.yml

```
@@ -453,14 +463,18 @@ jobs:
           # list of Docker images to use as base name for tags
           images: |
             nitnelave/lldap
+            ghcr.io/lldap/lldap
+          flavor:
+            suffix= ${{ matrix.suffix }}
```

```suggestion
          flavor: |
            latest=false
            suffix=${{ matrix.suffix }}
```
latest to false since we manually handle it